README.md

```diff
@@ -221,11 +246,12 @@ git push origin :<branch-name>
 
 ## Syncing a fork
 
-Connect your local repository to the original “upstream” repository by adding it as a remote. Pull in changes from “upstream” often so that you stay up to date so that when you submit your pull request, merge conflicts will be less likely. 
+Connect your local repository to the original “upstream” repository by adding it as a remote. Pull in changes from “upstream” often so that you stay up to date so that when you submit your pull request, merge conflicts will be less likely.
 
 Refer to [Sync a fork of a repository to keep it up-to-date with the upstream repository.](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork)
```

Comment:
Change to For more information, see...

That is the standard way IBM style suggests that we use for cross references.